### ISSUE TITLE
window borders sometimes wrong or don't show up

### ISSUE THREAD

**Issue by hertg** [leftwm/leftwm#735]
I experienced the following two issues in the past weeks.

**Some applications don't get the focused border color**
I found that some applications (eg. IntelliJ IDEA) don't get `focused_border_color` when focused, the border always stays the `default_border_color`. I don't know if it affects just IntelliJ, or applications written in Java in general.

~~**Border colors for floating/non-floating windows seem to get mixed up**
When I switch focus from a tiled window to a floating window, the previously focused tiled window gets the `floating_border_color`. When switching focus from a tiled to another tiled window, this doesn't happen. The floating window seems to never receive the `floating_border_color` at all.~~ 
=> fixed in #739 

**Environment:**
I'm on latest main
 ```text
:: LeftWM version: 0.3.0
:: LeftWM git hash: c3de436
:: Loading configuration . . .
    -> Configuration loaded OK
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK
:: Checking theme . . .
    -> Theme OK
 ```

**Comment by hertg** [comment]
I was able to fix the floating border-color issue, but the first issue with IntelliJ not getting the focused border color still persists. I found that the issue is here: https://github.com/leftwm/leftwm/blob/c3de436a3349b1b53dce5d774fe3328802621b88/leftwm-core/src/display_servers/xlib_display_server/xwrap/window.rs#L258

The IntelliJ window has `never_focus` set to true. I'm not really knowledgeable enough about why that is. I found that xprop returns the following WM_HINTS though. So `Client accepts input or input focus: False` seems weird.

```txt
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: False
		Initial state is Normal State.
```

Most likely related to IntelliJ being a Java AWT application, and probably related to #488.

---

I will open a PR to fix the second issue right away, it's just a one-line change.

**Comment by hertg** [comment]
I just fixed the second issue with #739, the first issue hasn't been fixed yet.

**Comment by VuiMuich** [comment]
Oh, I guess this was auto-closed by the merge..

**Comment by hertg** [comment]
closed via #761
